master/buildbot/schedulers/basic.py

```diff
@@ -46,7 +46,8 @@ class BaseBasicScheduler(base.BaseScheduler):
     class NotSet:
         pass
 
-    def __init__(self, name, shouldntBeSet=NotSet, treeStableTimer=None,
+    def __init__(self, name, shouldntBeSet=NotSet, 
+                 treeStableTimer=None, coolDownTimer=None,
```

Comment:
I wonder if there're instances that depend on the *order* of keyword parameters :/